WorkingInR/2_First_Steps_in_R.Rmd

```diff
@@ -241,20 +253,22 @@ class(comp)
 
 
 
-## Useful objects
+## Objects
 
 R can handle a multitude of object types, but you will certainly need to know only about the following ones: vector, factor, list, matrix, data frame and tibble.
 
```

Comment:
Data in R may be stored in a multitude of object types, but you only need to know a few of them to continue on this website: vector, matrix, list, data frame and tibble.

maybe change to:
...but the most important once are...